geniza/corpus/admin.py

```diff
@@ -295,29 +321,57 @@ def view_old_pgpids(self, obj):
         ("secondary_languages", admin.RelatedOnlyFieldListFilter),
     )
 
-    fields = (
-        ("shelfmark", "id", "view_old_pgpids"),
-        "shelfmark_override",
-        "doctype",
-        ("languages", "secondary_languages"),
-        "language_note",
-        "description",
+    # organize into fieldsets so that we can insert inlines mid-form
+    fieldsets = (
         (
-            "doc_date_original",
-            "doc_date_calendar",
-            "doc_date_standard",
-            "standard_date",
+            None,
+            {
+                "fields": (
+                    ("shelfmark", "id", "view_old_pgpids"),
+                    "shelfmark_override",
+                    "doctype",
+                    ("languages", "secondary_languages"),
+                    "language_note",
+                    "description",
+                )
+            },
+        ),
+        (
+            None,
+            {
+                "fields": (
+                    (
+                        "doc_date_original",
+                        "doc_date_calendar",
+                        "doc_date_standard",
+                        "standard_date",
+                    ),
+                ),
+            },
+        ),
+        (
+            None,
+            {
+                "fields": (
+                    "tags",
+                    "status",
+                    ("needs_review", "notes"),
+                    "image_order_override",
+                    "admin_thumbnails",
+                )
+            },
         ),
-        "tags",
-        "status",
-        ("needs_review", "notes"),
-        "image_order_override",
-        "admin_thumbnails",
         # edition, translation
     )
     autocomplete_fields = ["languages", "secondary_languages"]
     # NOTE: autocomplete does not honor limit_choices_to in model
-    inlines = [DocumentTextBlockInline, DocumentFootnoteInline, DocumentDatingInline]
+    inlines = [
+        DocumentDatingInline,
+        DocumentTextBlockInline,
+        DocumentFootnoteInline,
+        DocumentPersonInline,
+    ]
+    fieldsets_and_inlines_order = ("f", "f", "i", "f", "i", "i", "i")
```

Comment:
this would benefit from a little more context - suggest a one-line comment referencing the template where you have the link to the source